site/content/docs/5.3/components/side-navigation.md

```diff
@@ -0,0 +1,1425 @@
+---
+layout: docs
+title: Side Navigation
+description: Documentation and examples for Boosted's exclusive Brand responsive side navigation.
+group: components
+aliases:
+  - "/docs/components/side-navigation/"
+toc: true
+added: "5.3"
+---
+
+## How it works
+
+Our side navigation is basically a `<nav>` that provides a hierarchy between links.
+
+Side navigation should be used inside an offcanvas using `.offcanvas-*` for responsive purpose. By default, we use `.offcanvas-lg` in our docs. Please refer to [Offcanvas]({{< docsref "/components/offcanvas#responsive" >}}) to learn more about its behavior. Since it uses [responsive offcanvas]({{< docsref "/components/offcanvas#responsive" >}}), please resize your browser to see its responsive behavior in action.
+
+{{< callout warning >}}
+In order to make side navigation work properly, please make sure that the Javascript files from [Collapse]({{< docsref "/components/collapse#usage" >}}) and [Tabs]({{< docsref "/components/navs-tabs#javascript-behavior" >}}) are imported in your project.
+{{< /callout >}}
+
+The side navigation provides 3 ready-to-use fully responsive themes. For each theme, we provide:
+- `.side-nav` to set up at the root element.
+- `.side-nav-dark` to darken our theme variants.
+- `.side-nav-scrollable` to ensure correct behavior of scroll boxes.
+- `.side-nav-item` to ensure correct size and behavior of displayed elements.
+- optional `.side-nav-static` to make collapsible side navigation static.
+- optional `.side-nav-toggle` to embed a version of the to collapse toggler.
+- optional `.side-nav-content` to make sure the content is well sized even with a toggler button.
+
+With the following structure:
+
+```html
+<nav class="side-nav side-nav-dark">
+  <div class="side-nav-theme side-nav-static">
+    <!-- If collapsible -->
+    <button class="side-nav-toggle"></button>
+    <div class="side-nav-content">
+    <!-- End collapsible -->
+      <a href="#" class="side-nav-item">Links</a>
+    </div>
+  </div>
+</nav>
+```
+
+## Examples
+
+Side navigation come with 3 ready-to-use fully responsive themes. If these available themes don’t meet your specific needs, feel free to develop your own custom theme accordingly to the Orange Design System.
+
+Since most of side navigation we display use the responsive version of this component, please resize your browser to see the behavior.
+
+{{< callout info >}}
+You can also notice that every example is resizable in order to show the behavior it has in different configurations.
```

Comment:
Not sure if it need to be in a callout? Or if the callout should also include line 50?